.github/workflows/examples.yaml

```diff
@@ -15,14 +15,13 @@
 name: Examples
 
 on:
-    push:
-    pull_request:
+  push:
+  pull_request:
```

Comment:
It looks like the indentation used within the changes in this file are inconsistent with the rest of the indentation of the file

Reply:
Agree with this, please let's stick to the same indentation (note: if you use VSCode dev container, it should be consistent)

Reply:
Indentation should be fixed